spec/controllers/bookmarks_controller_spec.rb

```diff
@@ -42,4 +65,19 @@ def it_redirects_to_user_login
       end
     end
   end
+
+  describe "show" do
+    let(:chapteredwork) { create(:work) }
+    let(:chapter2) { create(:chapter, work: chapteredwork) }
+    let(:bookmark) { create(:bookmark, bookmarkable_id: chapteredwork.id) }
+    render_views
```

Comment:
We generally avoid `render_views` in our specs -- if something needs the view, we tend to prefer to use features. Is there a specific untested line in the controller you're trying to hit here?

Reply:
Will fix.